journal/boyington.tex

```diff
@@ -0,0 +1,878 @@
+Although useful, there exist limitations on the ammount of information that can be extracted from a neutron signal using an increased number of fission chambers.
+Geometric constraints and isotopic availability and manufacturing exist as two major limitations.
+Another issue, the problem considered in this particular work, deals with the transformation of response data into the spectral neutron flux, specifically, the underdetermined problem of spectral deconvolution caused by both the uncertainty in the responses and the disparity between the response vector and the desired, larger neutron spectral flux vector.
+Previously, a scoping study was completed demonstrating the performance of three deconvolution algorithms on this specific problem: minimum-norm, Tihkonov regularization, and maximum Shannon entropy unfolding \cite{roberts2018use}.
+These algorithms serve as a starting point for approaching this problem; however, they each present certain pathologies that motivate the search for better performing methods.
+The deconvolution algorithms MAXED and Gravel are two approaches presented in this work that address these specific issues and improve the overal error associated with the deconvoluted solution.
+
+
+\section{Spectral Deconvolution}
+
+Spectral deconvolution, also known as spectral unfolding, is a problem commonly found within neutron spectrometry.
+It arises in part by the desire to produces a high resolution representation of the neutron flux spectrum featuring useful detail in special regions.
+Consider an $N$-chambered device, each chamber containing a sufficient mass of a given fissionable isotope to produce a measureable signal, deployed in a reactor.
+If our goal is to produce a $g$-group structured representation of the flux spectrum, the system of equations governing the process, when represented as a summation, is expressed as follows:
+
+\begin{equation}
+  r_i + \epsilon_i = \sum_{g=1}^{N} \Sigma_{fg,i}  \phi_g\, , 
+    \qquad i = 1 \ldots N \, ,
+\label{eq:response}
+\end{equation}
+
+where each $r_i$ is the response of isotope $i$, $\epsilon_i$ is the error associated with $r_i$, $\Sigma_{fg,i}$ is the flux-weighted, integrated fission cross section for energy group $g$ for isotope $i$, and $\phi_g$ is the integrated flux in energy group $g$.
+Known in this equation are the responses $r_i$, obtained from detector measurements, and assumed is the matrix $\Sigma$.
+It is assumed because to flux-weight the cross section requires {\it a priori} knowledge of the true flux spectrum, the obtention of which is the goal of our work here.
+Deconvolution refers to the class of methods used to obtain the flux vector.
+
+Many different methods of deconvolution have been studied and shown to have varying levels of largely application-dependent success.
```

Comment:
I would insert the simpler schemes first by introducing an epsilon=0 approximation.  Then, build up the story to include the uncertainty and highlight how it is used by the more advanced algorithms.  (The challenge will be to define a suitable uncertainty)